packages/app-desktop/ElectronAppWrapper.ts

```diff
@@ -204,6 +205,23 @@ export default class ElectronAppWrapper {
 			}
 		});
 
+		ipcMain.on('minimize-handler', () => this.win_.minimize());
+
+		ipcMain.on('maximize-handler', () => this.win_.isMaximized() ?
+			this.win_.restore() : this.win_.maximize());
+
+		ipcMain.on('close-window-handler', () => this.win_.close());
+		ipcMain.on('display-app-menu', function(_: any, args: {x: number; y: number}) {
+			const menu = Menu.getApplicationMenu();
+			if (process.platform === 'win32') {
```

Comment:
What about other platforms? Same question for TitleBar.tsx

Reply:
The menu appears in Menu Bar at the top of the screen for other platforms

Reply:
I guess my question is, why are you checking for win32 here?

Reply:
To check if the app is being run on a Windows system

Reply:
What about Linux systems? They use the menu bar just like Windows. 

I see that you've now disabled this upgrade on non-windows systems so it's not necessary to change anymore.